HaulageApplication/Haulage/DatabaseSetup.cs

```diff
@@ -20,97 +20,177 @@ public static string GetDatabasePath()
+    public static void DropTables(SQLiteConnection connection) 
+    {
+        List<string> dropTableScripts = new List<string>();
+
+        dropTableScripts.Add("DROP TABLE IF EXISTS [User]");
+        dropTableScripts.Add("DROP TABLE IF EXISTS [Bill]");
+        dropTableScripts.Add("DROP TABLE IF EXISTS [Event]");
+        dropTableScripts.Add("DROP TABLE IF EXISTS [Expense]");
+        dropTableScripts.Add("DROP TABLE IF EXISTS [Item]");
+        dropTableScripts.Add("DROP TABLE IF EXISTS [Role]");
+        dropTableScripts.Add("DROP TABLE IF EXISTS [Trip]");
+        dropTableScripts.Add("DROP TABLE IF EXISTS [TripManifest]");
+        dropTableScripts.Add("DROP TABLE IF EXISTS [Vehicle]");
+
+        foreach (string table in dropTableScripts)
+        {
+            var command = new SQLite.SQLiteCommand(connection);
+            command.CommandText = table;
+            command.ExecuteNonQuery();
+        }
+    }
 
-                //Create User Table for Haulage Data;
+    private static void GenerateData(SQLiteConnection connection)
+    {
+        //Vehicle data
+        CreateVehicles(connection);
+    }
 
-                createTableScripts.Add(@"
+    private static void CreateVehicles(SQLiteConnection connection)
+    {
+        List<string> dataScripts = new List<string>();
+
+        dataScripts.Add(@"
+        INSERT INTO[Vehicle]
+           ([VehicleId]
+           , [tripID]
+           , [LicensePlate]
+           , [Capacity]
+           , [DriverId]
+           , [Status])
+             VALUES
+            (1, 1, 'test1', 1, 1,1);");
+        
+        dataScripts.Add(@"
+        INSERT INTO[Vehicle]
+           ([VehicleId]
+           , [tripID]
+           , [LicensePlate]
+           , [Capacity]
+           , [DriverId]
+           , [Status])
+             VALUES
+           (2,2, 'test2', 2, 2,1);");
+        
+        dataScripts.Add(@"
+        INSERT INTO[Vehicle]
+           ([VehicleId]
+           , [tripID]
+           , [LicensePlate]
+           , [Capacity]
+           , [DriverId]
+           , [Status])
+             VALUES
+           (3,3, 'test3', 3, 3,3);");
+
+        foreach (string tableScript in dataScripts)
+        {
+            var command = new SQLite.SQLiteCommand(connection);
+            command.CommandText = tableScript;
+            command.ExecuteNonQuery();
+        }
+    }
+
+    private static void CreateTables(SQLiteConnection connection)
+    {
+
+        List<string> createTableScripts = new List<string>();
+
+        //Create User Table for Haulage Data;
+
+        createTableScripts.Add(@" 
                 CREATE TABLE IF NOT EXISTS user (
                 UserID INTEGER PRIMARY KEY AUTOINCREMENT,
                 RoleID INTEGER NOT NULL,
                 FullName TEXT NOT NULL
                  );");
 
-                createTableScripts.Add(@"
+        createTableScripts.Add(@"
                 CREATE TABLE IF NOT EXISTS item (
```

Comment:
Naming conventions should be followed throughout, used lowercase for table name, used starting acpital followed by camel case. Please update to "Item".